usmqe_tests/web/tendrl/test_login.py

```diff
@@ -0,0 +1,89 @@
+"""
+Description: Simple tests for log in on tendrl UI
+
+Author: ltrilety
+"""
+
+
+import pytest
+from selenium.webdriver.common.keys import Keys
+
+from webstr.selenium.ui.support import WaitForWebstrPage
+from usmqe.web.tendrl.loginpage import pages as loginpage
+from usmqe.web.tendrl.mainpage.landing_page.pages import get_landing_page
+
+LOGGER = pytest.get_logger('login_test', module=True)
+
+
+def test_positive_login(testcase_set, testcase_end):
```

Comment:
> That said I wonder why the interest now.

I haven't noticed that before, sorry for lagging behind there.

> because there could be a test which doesn't want to close a browser

Why do we want session to remain open after the test case ends? I can't imagine a test which requires that, it could only break tests executed after that. What is the use case here? If you need that for debugging during development of tests, I would rather introduced configuration option for this rather than splitting fixtures in this way.

Anyway, this would not affect this particular pull request because as you said, it's there for some time already.